server/typedb.yml

```diff
@@ -0,0 +1,19 @@
+server:
+    address: 0.0.0.0:1729
+    http_enabled: true
+    http_address: 0.0.0.0:8000
+
+    authentication:
+        token_expiration_seconds: 5000
+
+    encryption:
+        enabled: false
+        # cert: =
+        # cert_key: =
+        # root_ca: =
```

Comment:
Not a fan of these fields, but they're what the keys for the CLI were.

Reply:
is there a way we can keep the 2.x model where all the keys exist, but are empty (then everyone can see all the possible options)

i think that's what we had right?